File: demo-app/app/src/main/java/com/launchkey/android/whitelabel/demo/ui/activity/CustomSecurityActivity.java
package com.launchkey.android.whitelabel.demo.ui.activity;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.widget.Toolbar;

import com.launchkey.android.authenticator.sdk.ui.SecurityFragment;
import com.launchkey.android.whitelabel.demo.R;

/**
 * Created by armando on 5/9/17.
 */

public class CustomSecurityActivity extends BaseDemoActivity {

    private SecurityFragment mSecFragment;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.demo_activity_security);

        final Toolbar toolbar = (Toolbar) findViewById(R.id.sec_toolbar);
        setSupportActionBar(toolbar);

        mSecFragment = (SecurityFragment) getSupportFragmentManager()
                .findFragmentById(R.id.sec_security);
    }
}

File: demo-app/app/src/kotlinApp/java/com/launchkey/android/authenticator/demo/ui/activity/CustomSecurityActivity.kt
package com.launchkey.android.authenticator.demo.ui.activity

import android.os.Bundle
import android.support.v7.widget.Toolbar

import com.launchkey.android.authenticator.demo.R
import com.launchkey.android.authenticator.sdk.ui.SecurityFragment

/**
 * Created by armando on 5/9/17.
 */

class CustomSecurityActivity : BaseDemoActivity() {

    private var mSecFragment: SecurityFragment? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.demo_activity_security)

        val toolbar = findViewById<Toolbar>(R.id.sec_toolbar)
        setSupportActionBar(toolbar)

        mSecFragment = supportFragmentManager
                .findFragmentById(R.id.sec_security) as SecurityFragment?
    }
}
